tests/packagedcode/data/docker/test-dockerfile/test.dockerfile-package.expected.json

```diff
@@ -0,0 +1,18 @@
+[

```

Comment:
This file and the file below have exactly the same data, you are not doing this correctly.

These are the two cases you want to test against:

1. The labels are being parsed correctly
2. The parsed data is being correctly mapped to a PackageData mapping

This should have had the fields of PackageData, mapped correctly, your tests look okay but it does not seem like you've run them or regenerated expectations.